Mathmagician.Tests/PrimeNumberTests.cs

```diff
@@ -19,8 +20,10 @@ public void MakeSureICanMakePrimeNumber()
         [TestMethod]
         public void CheckifPrime()
         {
+            
             PrimeNumber newprime = new PrimeNumber();
 
+            //This will test to see if "2" is a prime number.
             newprime.CheckIfNumberIsPrime(2);
 
             Assert.IsTrue(newprime.CheckIfNumberIsPrime(2));
```

Comment:
@complikatyed You're calling this method twice. Call `CheckIfNumberIsPrime` once and save the result to variable.